apps/src/Sound.js

```diff
@@ -75,7 +75,8 @@ Sound.prototype.play = function(options) {
     // Hook up on-ended callback, although browser support may be limited.
     this.playableBuffers[index].onended = function() {
       this.isPlayingCount--;
-      if (this.isPlayingCount === 0) {
+      if (this.isPlayingCount <= 0) {
```

Comment:
We were sometimes ending up with negative numbers here (I think because of how we use stopAllSounds() in the reset logic). Constraining this value to not go below 0 seems like the right behavior, and makes it so that `this.isPlaying_` is always accurate.